__tests__/Stats.specs.ts

```diff
@@ -0,0 +1,34 @@
+import { beforeEach, describe, it, expect, vitest } from 'vitest'
+import StatsBox from '../components/StatsBox.vue'
+import { mount } from '@vue/test-utils'
+import { createTestingPinia } from '@pinia/testing'
+import { Fetchstatus } from '../store/fetchstatus'
+/* eslint-disable require-jsdoc */
+
+describe('Stats', () => {
+  let wrapper: any = null
+  let status: any = null
+  beforeEach(() => {
+    const pinia = createTestingPinia({
```

Comment:
Right on, didn't know about this function

Reply:
Yeah  this creates a testing pinia store in vitest, so that we can access the stores properties.